resources/views/components/input/datepicker.blade.php

```diff
@@ -20,15 +20,4 @@ class="datepicker_{{$id}} validate @error($id) invalid @enderror"
 </div>
 @endif
 
-@push('scripts')
-    <script>
-        $(document).ready(function() {
-            $('.datepicker_{{$id}}').datepicker({
-                format: '{{$format}}',
-                firstDay: 1,
-                yearRange: {{$yearRange}},
-                showClearBtn: true
-            });
-        });
-    </script>
-@endpush
+@include('components.input.init.datepicker')
```

Comment:
Note: I also moved these to a different folder just to have all the init scripts out there, but did not thought it necessary to apply the same logic there as well.